ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -4946,3 +4946,11 @@ EFHLC:
   - high_exhaust_fan_speed_status
   implements:
   - OPERATIONAL
+
+PM:
+  description: "Particle Concentration monitoring."
+  is_abstract: true
+  uses:
+  - zone_particle_concentration_sensor
```

Comment:
air or water? you can use `particle_concentration_sensor` which already exists, or you can add one for `zone_air_particle_concentration_sensor`.